config/crd/bases/infrastructure.cluster.x-k8s.io_awsclusters.yaml

```diff
@@ -16,7 +16,6 @@ spec:
     listKind: AWSClusterList
     plural: awsclusters
     singular: awscluster
-  preserveUnknownFields: false
```

Comment:
I think this needs an update to the Makefile to ensure we don't clobber this: https://github.com/kubernetes-sigs/cluster-api/blob/546d56bea2b8cd9ac24d90ed7599d209dfe2ccc5/Makefile#L187